Runtime/CesiumGlobeAnchor.cs

```diff
@@ -185,9 +185,14 @@ public double4x4 localToGlobeFixedMatrix
         /// </remarks>
         public double3 longitudeLatitudeHeight
         {
-            get => this._georeference.ellipsoid.CenteredFixedToLongitudeLatitudeHeight(this.positionGlobeFixed);
+            get
+            {
+                this.UpdateGeoreferenceIfNecessary();
+                return this._georeference.ellipsoid.CenteredFixedToLongitudeLatitudeHeight(this.positionGlobeFixed);
```

Comment:
What if this CesiumGlobeAnchor isn't inside a CesiumGeoreference? In that case, `_georeference` will still be null after the `UpdateGeoreferenceIfNecessary` call. Obviously the globe anchor can't actually work in that scenario, but it shouldn't crash, either. Returning (0,0,0) would probably be fine.